datasetinsights/io/kfp_output.py

```diff
@@ -68,3 +76,15 @@ def write_metric(self):
         logger.debug(
             f"Metrics file {self.filename} saved at path:" f" {self.filepath}"
         )
+
+    def create_tb_visualization_json(self):
```

Comment:
Does it dump the metrics into a json file? Why don't we need it before?

Reply:
Yes. Previously, we didn't have tensorboard visualization with in kubeflow dashboard which is a pretty useful addition to have.